parlai/agents/hugging_face/hugging_face.py

```diff
@@ -12,14 +12,18 @@
 hugging_face/dialogpt`.
 """
 try:
-    import transformers  # noqa: F401
+    import transformers
 except ImportError:
     raise ImportError('Please run `pip install transformers`.')
 
 
+version = float('.'.join(transformers.__version__.split('.')[:2]))
```

Comment:
nit; Can we identify this as a global more distinctly somehow? So that we don't end up mixing this up with some other variable named "version". Perhaps just all caps (`VERSION`)?

Reply:
yeah i'll make it more explicit (`HF_VERSION`)